app/views/dashboard/users/show.html.haml

```diff
@@ -8,14 +8,18 @@
       .btn-group.mr-2{ role: 'group' }
         = button_to 'Delete', dashboard_user_path(@user), class: ' btn btn-outline-dark', method: :delete, data: { confirm: 'Are you sure?' }
   %br
-.col-lg-8.col-sm-12
+.col-12.col-md-9.col-lg-7
   = div_for(@user, class: 'card') do
     .card-header
       User detail
     .card-body
       %dl.row
-        %dt.col.text-md-right Email:
-        %dd.col= @user.email
+        %dt.col-sm-5.col-md-4.text-md-right Email:
+        %dd.col-sm-7.col-md-6= @user.email
       %dl.row
-        %dt.col.text-md-right Roles:
-        %dd.col= @user.roles.join(', ')
+        %dt.col-sm-5.col-md-4.text-md-right Roles:
+        %dd.col-sm-7.col-md-6= @user.roles.join(', ')
+      - if @user.location_ids.present?
+        %dl.row
+          %dt.col-sm-5.col-md-4.text-md-right Manageble Locations:
+          %dd.col-sm-7.col-md-6= @user.locations.map(&:name_en).join(', ')
```

Comment:
Maybe better to have a helper method to return `location_names`, so in the model, we don't coupling with the gem and presentation logic.